inc/server/class-form-server.php

```diff
@@ -0,0 +1,233 @@
+<?php
+/**
+ * Card server logic.
+ *
+ * @package ThemeIsle\GutenbergBlocks\Server
+ */
+
+namespace ThemeIsle\GutenbergBlocks\Server;
+
+/**
+ * Class Plugin_Card_Server
+ */
+class Form_Server {
+
+	/**
+	 * The main instance var.
+	 *
+	 * @var Form_Server
+	 */
+	public static $instance = null;
+
+	/**
+	 * Rest route namespace.
+	 *
+	 * @var Form_Server
+	 */
+	public $namespace = 'themeisle-gutenberg-blocks/';
+
+	/**
+	 * Rest route version.
+	 *
+	 * @var Form_Server
+	 */
+	public $version = 'v1';
+
+	/**
+	 * Initialize the class
+	 */
+	public function init() {
+		add_action( 'rest_api_init', array( $this, 'register_routes' ) );
+	}
+
+	/**
+	 * Register REST API route
+	 */
+	public function register_routes() {
+		$namespace = $this->namespace . $this->version;
+
+		register_rest_route(
+			$namespace,
+			'/forms',
+			array(
+				array(
+					'methods'             => \WP_REST_Server::READABLE,
+					'callback'            => array( $this, 'get_form_data' ),
+					'permission_callback' => function () {
+						return __return_true();
+					},
+				),
+			)
+		);
+
+		register_rest_route(
+			$namespace,
+			'/forms',
+			array(
+				array(
+					'methods'             => \WP_REST_Server::CREATABLE,
+					'callback'            => array( $this, 'get_form_data' ),
+					'permission_callback' => function () {
+						return __return_true();
+					},
+				),
+			)
+		);
+	}
+
+	/**
+	 * Search WordPress Plugin
+	 *
+	 * Search WordPress plugin using WordPress.org API.
+	 *
+	 * @param mixed $request Search request.
+	 *
+	 * @return mixed|\WP_REST_Response
+	 */
+	public function get_form_data( $request ) {
+
+		$return = array(
+			'success' => false,
+		);
+
+		$data = json_decode( $request->get_body(), true );
+
+		$email_subject = '[Otter] ' . ( isset( $data['emailSubject'] ) ? $data['emailSubject'] : __( 'A new submission', 'otter-blocks' ) );
```

Comment:
Default subject can be "A new form submission on (website name)," also remove "[Otter]" prefix.